### ISSUE TITLE
Publish --> MQTT Gateway very slow

### ISSUE THREAD

**Issue by Mickaelh51** [mysensors/Arduino#309]
Hi,
I'm using a dev branch with "GatewayW5100MQTTClient example" and node with DHT11 sensor + relay.

OpenHAB <--> Mosquitto <--> MySensors Gateway (id=0) <--> DHT11 + relay node (id=9)
- Openhab + mosquitto = the same server
- Gateway = arduino UNO + W5100 + NRF24L01
- Node = arduino UNO  + NRF24L01 + DHT11 + one relay

When I send a command from OpenHAB to node (id=9) this order is very very slow (5 to 10 seconds) to be executed in node id 9. But this "slow time" is randomly.

Example:
from openhab I send to MQTT server this order: mygateway1-in/9/1/1/0/2 1 (to activated the relay).
Sometimes it's fast, sometimes it's slow.
I traced MQTT protocol with tcpdump, I can see my arduino MQTT gateway  doesn't answer fast. Mosquitto must resend the same mqtt message.
When was fast, mosquitto send MQTT message and MQTT gateway answer very quickly.

but when node send new data from DHT11, these data arrive quickly to OpenHAB.

to recap:
1) DHT11 --> node 9 --> MQTT Gateway --> Mosquitto --> Openhab = very fast
2) Openhab -(fast)-> Mosquitto -(slow)-> MQTT Gateway -(fast)-> node 9 -(fast)-> relay 

Do you have an idea for this issue ?

thanks in advance
